css/main.css

```diff
@@ -98,8 +131,7 @@ body {
 }
 
 .button--large {
-  font-size: 18px;
-  height: 60px;
+  background: var(--c-main-primary);
```

Comment:
please revert changes, because currently what `.button-large` does is just changes a background color